src/PrestaShopBundle/Utils/ImmutableFloat.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace PrestaShopBundle\Utils;
+
+/**
+ * Immutable value-object that contains a float value.
+ */
+class ImmutableFloat
```

Comment:
IMVHO the `Immutable` part is tautological .. A float class is a VO, so it is obviously immutable. Avoiding `Immutable` makes it more compact, readable and easy to use .. If a MutableFloat would exists then I'll agree .. but having only one I think this is to be avoided